platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/PerformanceEvent.java

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.mapboxsdk.module.telemetry;
```

Comment:
if you are adding this class to the public API of telemetry defintion, you should place it under `module` as these files are removed by some clients to roll their own.

Note however that in a another comment, I'm indicating to remove this class from the public API so no action should be taken, just an FYI.

Reply:
@tobrun I felt that PerformanceEvent should not be public and either:
-  I should add public PerformanceEventFactory that creates those events 
- or keep it together with the impl classes. (i.e. in the module).